students/SLammertink/Lesson03/list_lab.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python3
+# Author: SLammertink
+# UW Self-paces Lesson 03 List_lab.py
+
+# Print a fruit list
+
+## Series 1 ##
+
+print(' ')
+print('Now we will print a list with the given fruits: ')
+
+fruit_list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(fruit_list)
+print(' ')
+# Ask the user to add a fruit to the list then print the list
+
+new_fruit = input("Which fruit would you like to add to the list: ")
+fruit_list.append(new_fruit)
+print(fruit_list)
+print(' ')
+
+# Ask the user to enter a number and print the fruit related to this number in the list:
+print('Now we ask the user to enter a number and print the fruit associated with that number in the list: ')
+print(' ')
+number_asked = int(input("Please enter a number:  "))
+for i, fruit in enumerate(fruit_list, 1):
+    if i == number_asked:
+        print("The number you entered was {} and the corresponding fruit is {}.".format(number_asked, fruit))
```

Comment:
This can be done without the for loop:
```
"The number you entered was {} and the corresponding fruit is {}.".format(number_asked, fruit_list[number_asked - 1])
```